msteams-platform/concepts/design/map-use-cases.md

```diff
@@ -1,72 +1,105 @@
 ---
-title: Map your use cases to app capabilities
+title: Map your use cases to Teams app capabilities
 author: clearab
-description: Decide how to distribute your app
+description: Choose the correct app scope
 ms.topic: conceptual
 ms.author: anclear
 ---
-# Map your use cases to teams app capabilities
 
-If you haven't already, make sure you've [considered your use cases](~/concepts/design/map-use-cases.md) carefully. You should also have a good understanding of the [extensibility points and UI elements](~/concepts/extensibility-points.md) available for your app. Once you've figured out *what* your trying to solve, and *who* you're solving it for, it is time to start thinking about *how*.
+# Map your use cases to Teams app capabilities
 
-Below you'll find some common scenarios, and a selection of extensibility points and UI elements that work well with them. It isn't intended to be an exhaustive list, just to help you think through some of the possibilities available to you and the Teams platform.
+After you have identified *who* the user is and *what* problem you will solve, it is time to decide *how* to solve the problem. The *who*, *what*, and *how* completes the process of understanding and mapping your use cases to Teams app capabilities. You need to define the scope of the app based on the responses you have received and then decide which capability is best suited to build your app.
```

Comment:
"...based on responses you have received..." - What responses are these? May be I am missing some context

Reply:
... responses received from the user to your queries